ios/browser/api/certificate/utils/brave_certificate_x509_utils.h

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_CERTIFICATE_UTILS_BRAVE_CERTIFICATE_X509_UTILS_H_
+#define BRAVE_IOS_BROWSER_API_CERTIFICATE_UTILS_BRAVE_CERTIFICATE_X509_UTILS_H_
+
+#include <string>
+#include <vector>
+
+#include "base/memory/ref_counted.h"
+#include "base/time/time.h"
+#include "net/cert/internal/signature_algorithm.h"
+#include "net/cert/signed_certificate_timestamp.h"
+#include "net/der/input.h"
+#include "net/der/parse_values.h"
+#include "third_party/boringssl/src/include/openssl/base.h"
+
+namespace x509 {
```

Comment:
the top-level namespaces should match the "module" which in this case would be `certificate`

Reply:
Fixed.